pages/linux/vgs.md

```diff
@@ -0,0 +1,32 @@
+# vgs
+
+> Display information about volume groups.
```

Comment:
Perhaps this would read better?
```suggestion
> Display information about LVM volume groups.
```

Thanks for this trio of pages - they make a great addition to tldr-pages :D